Week2/home_work/Part1.js

```diff
@@ -0,0 +1,60 @@
+const mysql = require('mysql');
+const util = require('util');
+
+var connection = mysql.createConnection({
+   host: 'localhost',
+   user: 'hyfuser',
+   password: 'hyfpassword',
+   database: 'new_world'
+});
+
+const execQuery = util.promisify(connection.query.bind(connection));
+
+async function world() {
+
+   function countryCapital(x) {
+      const result = `SELECT country.name AS city, city.name AS capital FROM city, country WHERE id = capital AND country.name = ${x} `
```

Comment:
You can use the "escape" functionality provided by mysql.js to protect yourself from malicious user input https://github.com/mysqljs/mysql#escaping-query-values